content/ja/handbook/engineering/development/growth/index.md

```
@@ -0,0 +1,139 @@
+
+[バリュー](/handbook/values/) に沿って、[イテレーション](/handbook/values/#iteration) と [コラボレーション](/handbook/values/#collaboration) に焦点を当て、開発部門のカウンターパートが管理するプロダクトの領域と協力して取り組んでいます。
+
+プロダクトチームが優先するIssueに取り組み、GitLab.com での [実験](/handbook/engineering/development/growth/experimentation/) も実行しています。Growth ステージのチームにはフルスタックエンジニアがいます。その理由は、Growth ステージはフロントエンドとバックエンドの両方のスキルセットを必要としますが、小規模なチームとして、チームメンバーの効率を最適化するためにフルスタックロールを採用しているからです。
+
+## Growth リーダーシップとチーム
+
+### リーダーシップ
+
+| ロール | 人 |
+|------|--------|
+| プロダクトディレクター | 
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)